csm/cli/conf/etc/cli/cortxcli.conf

```diff
@@ -22,6 +22,8 @@ Log:
     #file_size: 10
     log_path: "/var/log/seagate/csm/"
     max_result_window: 10000
+    uds_log_path: "/var/log/uds"
+    elasticsearch_log_path: "/var/log/elasticsearch/elasticsearch_cluster.log"
```

Comment:
We should get other log files from elasticsearch too.
1. elasticsearch_cluster_index_indexing_slowlog.log
2. elasticsearch_cluster_index_search_slowlog.log
3. gc.log.0.current

Reply:
added additional files